packages/sdk-communication-layer/src/RemoteCommunication.ts

```diff
@@ -718,23 +717,17 @@ export class RemoteCommunication extends EventEmitter2 {
         );
       }
 
-      // FIXME remove after backward compatibility
-      // backward compatibility for wallet <6.6
+      // TODO remove after wallet 7.3+ is deployed
+      // backward compatibility for wallet <7.3
       if ('7.3'.localeCompare(this.walletInfo?.version || '') === 1) {
```

Comment:
we shouldn't check for a version and create backward compatibility to a version doesn't exist.
- 7.3 could not exist
- the fix on the wallet that this is supposed to work with isn't merged yet